pyccel/ast/core.py

```diff
@@ -4491,6 +4537,14 @@ class IfTernaryOperator(If):
 
     pass
 
+class StarredArguments(Basic):
+    def __new__(cls, args):
+        return Basic.__new__(cls, args)
+
+    @property
+    def args(self):
```

Comment:
See earlier comment about overriding `Basic.args`

Reply:
This also seems to be true in the class `Concatenate`.
Does anyone have a suggestion for a better property name there? @saidctb ?

Reply:
Assuming that `Concatenate` is used for String concatenation as the docstring suggests, I am not sure if it is ever used in Pyccel. I can find a single place in Pyccel where an object of this class is created, it's in `SemanticParser._visit_Add()`: https://github.com/pyccel/pyccel/blob/54125929d992a54b2d7739b7c4a3827381977de0/pyccel/parser/semantic.py#L1519

(@saidctb: Can you tell us when and how `Concatenate` is used?)